documentation/running-services/running-with-docker.md

```diff
@@ -8,6 +8,17 @@ title: Running services with Docker
 
 The [hocs](https://github.com/UKHomeOffice/hocs) repository contains a [Docker Compose](https://docs.docker.com/compose/) file in `hocs/docker/docker-compose.yml`.
 
+### Contents:
```

Comment:
@robertdeniszczyc2 I think you can just use `[[toc]]` here - not sure what level headers it picks up though. A little better because it means if headings change names or get added / removed the references are updated.

Reply:
Ah! That's what I was looking for, I was sure I'd seen an automatic thing before. I'll raise a new PR to update it to that

Reply:
https://github.com/UKHomeOffice/hocs/pull/67